### PR TITLE
Feature/alex updates

### PR DESCRIPTION
**Pull request by alexcolbeck1**
## Checklist
- [ ] Design and code examples added to the Design System pages
- [ ] Changes added to [CHANGELOG.md](../CHANGELOG.md)
- [ ] Successful linting of the SCSS (`npm run lint-css`)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,6 @@ bower_components
 Thumbs.db
 
 dist/css/styles.css
+*.json
+package.json
```
Comment (chrimesdev) [inline .gitignore]: We need to keep package.json in the Git repository, as this is where all the packages and project information is stored. Without it people wouldn't be able to install the required dependencies or run the project.

File: .gitignore [hunk 2]
```diff
@@ -30,3 +30,6 @@ bower_components
...
+package-lock.json
```
Comment (chrimesdev) [inline .gitignore]: We need to keep the package-lock.json in the git repository too. See the comment on here https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

File: .gitignore [hunk 3]
```diff
@@ -30,3 +30,6 @@ bower_components
 Thumbs.db
 
 dist/css/styles.css
+*.json
```
Comment (chrimesdev) [inline .gitignore]: See the comments above

File: nhsuk-design-system/views/includes/header-v1/site-layout.html [hunk 4]
```diff
@@ -21,7 +21,8 @@
     <link rel="stylesheet" href="/css/prism.css">
 
     <script src="/js/clipboard.min.js"></script>
-
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
Comment (chrimesdev) [inline nhsuk-design-system/views/includes/header-v1/site-layout.html:24]: When including externally hosted resources make sure you include Subresource Integrity checking.

> The integrity and crossorigin attributes are used for Subresource Integrity (SRI) checking. This allows browsers to ensure that resources hosted on third-party servers have not been tampered with. Use of SRI is recommended as a best-practice, whenever libraries are loaded from a third-party source. Read more at srihash.org

Example:

`<script
  src="https://code.jquery.com/jquery-3.3.1.min.js"
  integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
  crossorigin="anonymous"></script>`

File: nhsuk-design-system/views/includes/site-layout.html [hunk 5]
```diff
@@ -20,6 +20,7 @@
     <link rel="stylesheet" href="/css/styles.css">
 
     <script src="/js/clipboard.min.js"></script>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
Comment (chrimesdev) [inline nhsuk-design-system/views/includes/site-layout.html]: See comment above

**Review by chrimesdev — CHANGES_REQUESTED** [review]

**Comment by alexcolbeck1** [comment]
Changes made

**Review by chrimesdev — APPROVED** [review]

**Comment by chrimesdev** [comment]
Cheers, looks good!

**Pull request merged**